extensions/git/src/commands.ts

```diff
@@ -1932,7 +1944,7 @@ export class CommandCenter {
 						return Promise.resolve();
 					}
 
-					return Promise.resolve(method.apply(this, [repository, ...args]));
+					return Promise.resolve(method.apply(this, [repository, ...args.slice(1)]));
```

Comment:
Same change as in #58963.
Allows to pass multiple variables in case of `repository` being set to true in the command options.

Reply:
Reverted this, since it is unnecessary after the refactoring.